src/graphql/utils/getPageInfo.ts

```diff
@@ -0,0 +1,18 @@
+import { DEFAULT_LIMIT } from '../config'
+
+export default function getPageInfo(
+  totalCount: number,
+  page: number = 1,
+  limit: number = DEFAULT_LIMIT,
+) {
+  // Calculate the page info details
+  const totalPages = Math.round(totalCount / limit)
```

Comment:
If we have `31` items and a limit of `10` this would result in the total amount of pages to be `3` when it should actually be `4` to hold the one extra item?

Perhaps we should replace this with `Math.ceil` instead?

Reply:
Yes, good one